packages/console/src/pages/EnterpriseSsoDetails/Experience/DomainsInput/index.tsx

```diff
@@ -32,12 +32,14 @@ function DomainsInput({ className, values, onChange: rawOnChange, error, placeho
   const [focusedValueId, setFocusedValueId] = useState<Nullable<string>>(null);
   const [currentValue, setCurrentValue] = useState('');
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
-  const { setError } = useFormContext<DomainsFormType>();
+  const { setError, clearErrors } = useFormContext<DomainsFormType>();
 
   const onChange = (values: Option[]) => {
     const { values: parsedValues, errorMessage } = domainOptionsParser(values);
     if (errorMessage) {
       setError('domains', { type: 'custom', message: errorMessage });
+    } else {
+      clearErrors('domains');
```

Comment:
nit: You can always `clearErrors` first on value change.

Reply:
We have different logic here for the email domain input field, since the API only returns one error message and a real-time check on the front end shows more informative error info.

Reply:
If we clear the error message on change, the input component validator will hence throw a new error message with the same content. The console page could flash when the user clicks on the "save" button.